lib/PhasarLLVM/Passes/GeneralStatisticsAnalysis.cpp

```diff
@@ -244,6 +252,8 @@ llvm::raw_ostream &operator<<(llvm::raw_ostream &OS,
             << "LLVM IR instructions:\t" << Statistics.Instructions << "\n"
             << "Functions:\t" << Statistics.Functions << "\n"
             << "Global Variables:\t" << Statistics.Globals << "\n"
+            << "Global Variable Consts:\t" << Statistics.GlobalConsts << "\n"
+            << "Global Pointers:\t" << Statistics.GlobalPointers << "\n"
```

Comment:
This seems to be redundant. All globals are of pointer type by construction

Reply:
agreed. however, the code to set these counters where here before and I didn't want to remove it because I was not sure if someone uses it with exactly this output. 
```
    if (Global.getType()->isPointerTy()) {
      ++Stats.GlobalPointers;
    }
    ++Stats.Globals;
```

We can definitely remove this but I didn't want to sneak it in here to keep it clear, I just added it to the output to make it more transparent that this exists.